media/js/firefox/new/scene1-batm.js

```diff
@@ -5,11 +5,16 @@
 (function($) {
     'use strict';
 
-    var exp = $('#main-header-copy').data('experience');
+    var title = $('#main-header-copy');
+    var exp = title.data('experience');
+    var variant = title.data('variant');
 
     $('.download-link').each(function(i, link) {
-        if (exp && link.href.indexOf('scene=2') > -1) {
-            // specify v=1 template for scene 2
+        if (exp && variant && link.href.indexOf('scene=2') > -1) {
+            // append experience and variant
+            link.href = link.href.replace('scene=2', 'scene=2&xv=' + exp + '&v=' + variant);
+        } else if (exp && !variant && link.href.indexOf('scene=2') > -1) {
+            // append experience
             link.href = link.href.replace('scene=2', 'scene=2&xv=' + exp);
```

Comment:
Yeah, I think we could be a little clearer and more performant with something like:

```
var newParams = 'scene=2';

// conditionally construct new params for download links
if (exp) {
    newParams += '&xv=' + exp;
}

if (variant) {
    newParams += '&v=' + variant;
}

$('.download-link').each(function(i, link) {
    link.href = link.href.replace('scene=2', newParams);
});
```